core/src/com/xstudios/salvage/game/GameController.java

```diff
@@ -909,9 +917,10 @@ public void update(float dt) {
                 InputController input = InputController.getInstance();
                 if (input.isPause()) {
                     resume();
-                } else {
-                    updatePlayingState();
                 }
+//                else {
```

Comment:
We should uncomment this if we want the diver to be able to move during the pause screen, but I don't think that should be the case